package/com.unity.formats.usd/Runtime/Scripts/Timeline/UsdRecorderBehaviour.cs

```diff
@@ -145,14 +145,14 @@ public class UsdRecorderBehaviour : PlayableBehaviour {
       }
 
       Clip.Context = new ExportContext();
-      Clip.UsdScene.EndTime = currentTime * kExportFrameRate ;
+      Clip.UsdScene.EndTime = currentTime * kExportFrameRate;
 
       try {
         if(Clip.IsUSDZ && usdzTemporaryDir != null)
           Directory.SetCurrentDirectory(usdzTemporaryDir.FullName);
 
         Clip.Context = new ExportContext();
-        Clip.UsdScene.EndTime = currentTime;
+        Clip.UsdScene.EndTime = currentTime * kExportFrameRate;
```

Comment:
Looks like something was messed up during the PR merge and the ExportContext creation is now done twice. Can you remove the l147-148 please.

Reply:
Done. Turns out the kExportFrameRate error was also a merge issue probably, as that was also double (removed it as well).

For my understanding, why is the Clip.Context re-created in StopRecording at all?

Reply:
No, seems suspicious to me...